main.go

```diff
@@ -195,9 +197,38 @@ func main() {
 		reporter.Report()
 	}
 
+	accessKey := os.Getenv("ANODOT_ACCESS_KEY")
+	ifSendToBC := defaultIfBlank(os.Getenv("ANODOT_SEND_TO_BC"), "true")
+	sendToBCPeriod, err := strconv.Atoi(defaultIfBlank(os.Getenv("ANODOT_SEND_TO_BC_PERIOD_SEC"), "60"))
+	if err != nil {
+		log.Fatalf("Could not parse ANODOT_SEND_TO_BC_PERIOD_SEC: %v", err)
+	}
+	if ifSendToBC != "false" && len(strings.TrimSpace(accessKey)) > 0 {
+		sendToBC(primaryUrl, accessKey, token, sendToBCPeriod)
+	}
+
 	s.InitHttp(allWorkers)
 }
 
+func sendToBC(primaryUrl *url.URL, accessKey string, token string, sendToBCPeriod int) {
```

Comment:
rename to `sendAgentStatusToBC`..the reason i want to rename is to clearly mention that it is status sending :)